engine/src/main/java/org/terasology/rendering/nui/widgets/UIButtonWebBrowser.java

```diff
@@ -30,144 +30,162 @@
-                } else {
-                    runtime.exec(createCommand("xdg-open", this.url));
-                }
-            } catch (IOException e) {
-                LOGGER.warn("Can't recognize your OS and open the url {}.", this.url);
-                showErrorPopup("Can't recognize your OS and open the url " + this.url);
-            }
-        }
-    };
-
-    public UIButtonWebBrowser() {
-        this.url = "";
-
-        Config config = CoreRegistry.get(Config.class);
-        if (config != null) {
-            this.webBrowserConfig = config.getWebBrowserConfig();
-        }
-
-        this.subscribe(openUrlInDefaultBrowser);
-    }
-
-    /**
-     * Does confirmation and activates default action.
-     */
-    private void openBrowser() {
-        confirmed.set(true);
-        openUrlInDefaultBrowser.onActivated(this);
-    }
-
-    private void showConfirmationPopup() {
-        if (nuiManager == null || translationSystem == null) {
-            LOGGER.error("Can't show confirmation popup!");
-            return;
-        }
-
-        ConfirmUrlPopup confirmUrlPopup = nuiManager.pushScreen(ConfirmUrlPopup.ASSET_URI, ConfirmUrlPopup.class);
-        confirmUrlPopup.setMessage(translationSystem.translate("${engine:menu#button-web-browser-confirmation-title}"),
-                translationSystem.translate("${engine:menu#button-web-browser-confirmation-message}") + NEW_LINE + this.url);
-        confirmUrlPopup.setLeftButton(translationSystem.translate("${engine:menu#dialog-yes}"), this::openBrowser);
-        confirmUrlPopup.setRightButton(translationSystem.translate("${engine:menu#dialog-no}"), () -> { });
-        confirmUrlPopup.setCheckbox(webBrowserConfig, this.url);
-    }
-
-    private void showErrorPopup(final String message) {
-        if (nuiManager != null) {
-            nuiManager.pushScreen(MessagePopup.ASSET_URI, MessagePopup.class).setMessage("", message);
-        }
-    }
-
-    private boolean hasConfirmation() {
-        return confirmed.get() != null && confirmed.get();
-    }
-
-    private String[] createCommand(String... arguments) {
-        return arguments;
-    }
-
-    /**
-     * Sets the {@link UIButtonWebBrowser#url} value and sets {@link UIButtonWebBrowser#confirmed}
-     * to true if the given URL is already trusted, otherwise confirmed is false.
-     */
-    public UIButtonWebBrowser setUrl(String url) {
-        boolean trustedUrl = false;
-        if (webBrowserConfig != null) {
-            trustedUrl = webBrowserConfig.isUrlTrusted(url);
-        }
-
-        confirmed.set(trustedUrl);
-        this.url = url;
-        return this;
-    }
-
-    public UIButtonWebBrowser setNuiManager(final NUIManager nuiManager) {
-        this.nuiManager = nuiManager;
-        return this;
-    }
-
-    public UIButtonWebBrowser setTranslationSystem(final TranslationSystem translationSystem) {
-        this.translationSystem = translationSystem;
-        return this;
-    }
```

Comment:
Once again, you've intented this whole file and I don't think it was supposed to be. Try running the autoformat on the files before you commit them